config/services.php

```diff
@@ -105,4 +139,19 @@ function service(string $id): ReferenceConfigurator
             service(StreamedResponseListener::class.'.inner'),
             '%env(default::RR_MODE)%',
         ]);
+
+    $services
+        ->set(WorkflowClientInterface::class, WorkflowClient::class)
+        ->args([
+            service(ServiceClient::class)
+        ])
+    ;
+
+    $services
+        ->set(ServiceClient::class)
+        ->factory([ServiceClient::class, 'create'])
+        ->args([
+            'localhost:7233'
```

Comment:
This never change? `EnvironmentInterface` does not expose this socket?

Reply:
EnvironmentInterface provides RelayAddress. I'll use it.

Reply:
I can't be used. I made new parameter.